blocks/header/header.js

```diff
@@ -132,6 +133,32 @@ export default async function decorate(block) {
       });
     }
 
+    // language-selector
+    const pathName = window.location.pathname;
+    const currentLang = LANG_REGEX.exec(pathName)[0];
+    const navLanguage = nav.querySelector('.nav-language');
+    const languages = navLanguage.querySelectorAll('a');
+
+    languages.forEach((l) => {
+      const lang = LANG_REGEX.exec(l.href)[0];
+      if (currentLang === lang) {
+        const langSwitcher = document.createElement('div');
+        langSwitcher.classList.add('nav-language-switcher');
+
+        const langLabel = document.createElement('div');
+        langLabel.innerHTML = l.innerHTML;
+
+        const langPic = l.parentElement.parentElement.parentElement.querySelector('picture');
```

Comment:
what happens if the author forgot the pic? we should gracefully handle possible undefineds